opt_einsum/backends/dispatch.py

```diff
@@ -122,6 +123,7 @@ def has_tensordot(backend: str) -> bool:
     "cupy": _cupy.build_expression,
     "torch": _torch.build_expression,
     "jax": _jax.build_expression,
+    **_array_api.build_expression,
```

Comment:
Is there any way to check if `Jax` adds an array interface so that we can catch it and test before hand?